crossbeam/experiment/train_eval.py

```diff
@@ -218,6 +219,13 @@ def local_task_gen(domain):
       os.makedirs(log_folder)
     log_writer = tensorboard.SummaryWriter(log_folder)
   model = model.to(device)
+  optimizer = torch.optim.Adam(model.parameters(), lr=args.lr)
+  if checkpoint is not None:
+    model.load_state_dict(checkpoint['model'])
+    optimizer.load_state_dict(checkpoint['optimizer'])
+    starting_step = checkpoint['step']
```

Comment:
Can we skip `starting_step` batches from the training dataset, so that we don't train on the same data again upon restart? I think this is just:

```
for _ in range(starting_step):
  next(train_gen)
```